packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -61,7 +62,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         { ...{ onBlurCapture: this._onTooltipMouseLeave } }
         onMouseEnter={ this._onTooltipMouseEnter }
         onMouseLeave={ this._onTooltipMouseLeave }
-        aria-describedby={ isTooltipVisible ? tooltipId : undefined }
+        aria-describedby={ (isTooltipVisible && !stopAriaDescribedBy) ? tooltipId : undefined }
```

Comment:
@petdun2519, its not the case. ToolTipHost is a wrapper component and it sets `aria-describedby` on itself, which will not be read by screen reader. Check the GIF below:

![tooltiphost](https://cloud.githubusercontent.com/assets/12109362/26461523/fc159eb2-419a-11e7-9e2c-615fe09e4e24.gif)